### PR TITLE
Shawn Malluwa Proposal

### PR DESCRIPTION
**Pull request by Sheepy3**
## Describe your project in 1-3 sentences

me and two friends will be completing 3 projects. First, a CNC machine. Second, two USB macropads. finally, a budget PC. we will work on all these projects in parallel so as to complete as much as possible when together or apart. 

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by jaaldecrga** [comment]
Hello, this is Jaylan Garcia. I am confirming that I am working on this.
[X] I am a current high school student (or younger).
[X] I have submitted an RSVP at https://hackclub.com/winter
[X] I am a member of the Hack Club Slack (Join at https://hackclub.com/slack)
[X] I have self-reviewed my project proposal.https://github.com/hackclub/winter/pull/141

**Comment by CleiftonM** [comment]
hello, this is Cleifton Mertus. I am confirming that I am working on this.
[x] I am a current high school student (or younger).
[x] I have submitted an RSVP at https://hackclub.com/winter
[x] I am a member of the Hack Club Slack (Join at https://hackclub.com/slack)
[x] I have self-reviewed my project proposal.https://github.com/hackclub/winter/pull/141

**Review by bellesea — CHANGES_REQUESTED** [review]
@Sheepy3 thank you so much for submitting this PR and sharing your projects :))) Super cool that you're working together with your friends too hehe :)

I'm wondering, for the CNC machine, 
- where are you going to purchase your metal tubes? do you know which size to get (leep in mind inner diameter vs outer diameter)?
- Where is the CNC going to live afterwards?/ who is keeping it?
- When do you plan to start printing the parts? 

thank you sm!

**Comment by Sheepy3** [comment]
> @Sheepy3 thank you so much for submitting this PR and sharing your projects :))) Super cool that you're working together with your friends too hehe :)
> 
> I'm wondering, for the CNC machine,
> 
> * where are you going to purchase your metal tubes? do you know which size to get (leep in mind inner diameter vs outer diameter)?
> * Where is the CNC going to live afterwards?/ who is keeping it?
> * When do you plan to start printing the parts?
> 
> thank you sm!

1. Apologies for not clarifying, but we will be purchasing the metal tubing locally out of pocket, as well as wood for the custom PC case. I have confirmed the correct size and amount of length required.  
2. The CNC will live in my garage, and as we all live near each other, any of us can use it for future projects.
3. I plan to start printing parts for the CNC machine as soon as the filament arrives following approval. This is because of the sheer amount of parts needed, as I'm not sure ill be able to finish printing everything in time for feburary. I'll also be printing components for the macropads and PC case out of a significantly less rigid filament that I own as soon as we have designed all of the parts.

**Comment by bellesea** [comment]
> 1. Apologies for not clarifying, but we will be purchasing the metal tubing locally out of pocket, as well as wood for the custom PC case. I have confirmed the correct size and amount of length required.
no worries at all! this sounds good! could you please add to the file the size of the cnc machine you plan to build as well as the materials you'll be purchasing locally?

> 2. The CNC will live in my garage, and as we all live near each other, any of us can use it for future projects.
awesome!!

> 3. I plan to start printing parts for the CNC machine as soon as the filament arrives following approval. This is because of the sheer amount of parts needed, as I'm not sure ill be able to finish printing everything in time for feburary. I'll also be printing components for the macropads and PC case out of a significantly less rigid filament that I own as soon as we have designed all of the parts.
sounds good, can you please add this to your plan?

File: Shawn Malluwa.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
+---
+name: "@Shawn Malluwa"
```
Comment (bellesea) [inline Shawn Malluwa.md:2]:
```suggestion
name: ["@Shawn Malluwa", "@D3thclaw", "@jaaldecrga"]
```

**Review by bellesea — APPROVED** [review]
@Sheepy3 @D3thclaw @jaaldecrga this looks sooo awesome!! can't wait to see y'all work together on this :))

a fabulous member of @hackclub/bank-ops will be reaching out soon with next steps!

**Review by hackliv — APPROVED** [review]
Hi @Sheepy3 , @D3thclaw & @jaaldecrga! SO excited about this project. I'm dming you all on Slack now to chat about next steps for getting the grant through Hack Club Bank!

**Pull request merged**